scripts/audit.js

```diff
@@ -18,7 +18,11 @@ function npmAudit (pathname) {
   if (fs.existsSync(path.join(pathname, 'package.json')) &&
     fs.existsSync(path.join(pathname, 'package-lock.json'))) {
     console.log('Auditing', pathname)
-    util.run('npm', ['audit'], { cwd: pathname })
+    let cmdOptions = {
+      cwd: pathname,
+      shell: process.platform === 'win32' ? true: false
```

Comment:
minor: this should be `true :` instead of `true:`

i assume this doesn't work on windows without this change?